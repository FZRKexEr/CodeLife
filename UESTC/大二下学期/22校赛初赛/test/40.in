10 20
B 1
A 4 8 1
A 3 4 6
A 9 10 1
A 9 10 7
A 5 7 10
A 3 9 8
B 8
A 5 7 1
A 9 10 3
B 1
B 2
B 3
B 4
B 5
B 6
B 7
B 8
B 9
B 10
