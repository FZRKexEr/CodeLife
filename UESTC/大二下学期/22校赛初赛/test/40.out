1 10
8 8
1 2
1 2
3 3
4 4
5 7
5 7
5 7
8 8
9 9
10 10
